test/workbook/stakeholder-sheet.test.ts

```diff
@@ -314,4 +314,76 @@ describe(StakeholderSheet, () => {
     expect(excel.worksheets[0].getCell("M3").formula).toBe("=L3 / $L$8");
     expect(excel.worksheets[0].getCell("M2").value).toBe("Fully Diluted %");
   });
+
+  test("rounding method", () => {
+    const excel = new Excel.Workbook();
+    const workbookWriter = new ExcelJSWriter(excel);
+    const worksheetWriter = workbookWriter.addWorksheet("test");
+
+    const stockClasses = Array.of(
+      {
+        display_name: "Class A Common Stock",
+        is_preferred: false,
+      },
+      {
+        display_name: "Class A Preferred Stock Normal",
+        is_preferred: true,
+        conversion_ratio: 1,
+        rounding_type: "NORMAL",
+      },
+      {
+        display_name: "Class A Preferred Stock Normal",
+        is_preferred: true,
+        conversion_ratio: 2,
+        rounding_type: "NORMAL",
+      },
+      {
+        display_name: "Class A Preferred Stock Floor",
+        is_preferred: true,
+        conversion_ratio: 1.3,
+        rounding_type: "FLOOR",
+      },
+      {
+        display_name: "Class A Preferred Stock Ceiling",
+        is_preferred: true,
+        conversion_ratio: 0.7,
+        rounding_type: "CEILING",
+      }
+    );
```

Comment:
**Non-blocking:** There's a lot more setup here than is necessary to test this; the purpose of the test is obfuscated by all of the unnecessary classes.

I think this test could have safely lived in a much smaller more focused test in `holdings-column.test.ts`. It isn't particularly relevant to the larger "sheet".